frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.ts

```diff
@@ -0,0 +1,134 @@
+import {Location} from '@angular/common';
+import {Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {ActivatedRoute} from '@angular/router';
+import {isNullOrUndefined} from 'util';
+import {PhysicalPrintService} from '../../../services/physical-print/physical-print.service';
+import {PhysicalPrint} from '../../../model/physical-print';
+import {ErrorService} from '../../../services/error.service';
+
+@Component({
+  selector: 'app-physical-print-edit',
+  providers: [PhysicalPrintService, ErrorService],
+  templateUrl: './physical-print-edit.component.html',
+  styleUrls: ['./physical-print-edit.component.scss']
+})
+export class PhysicalPrintEditComponent implements OnInit {
+
+  @Input('physicalPrint') physicalPrint: PhysicalPrint = null;
+  @Input('nav') nav = true;
+  @Input('creating') creating = false;
+  @Output() changed: EventEmitter<PhysicalPrint> = new EventEmitter<PhysicalPrint>();
+  private loaded = false;
+
+  /* forms */
+  private requiredFieldsForm: FormGroup = null;
+
+  constructor(private route: ActivatedRoute,
+              private physicalPrintService: PhysicalPrintService,
+              private formBuilder: FormBuilder,
+              private errorService: ErrorService,
+              private _location: Location) {
+  }
+
+  ngOnInit(): void {
+    this.route.params.subscribe((params) => {
+      const id = params['id'];
+      if (id == 'create') { //new product is being created
+        this.create();
+      }
+      else if (id)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') if statements must be braced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS121)